MLTAscend/MLTAscend.Data/Helpers/PredictionHelper.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.EntityFrameworkCore;
+using dom = MLTAscend.Domain.Models;
+using System.Linq;
+
+
+namespace MLTAscend.Data.Helpers
+{
+    public class PredictionHelper
+    {
+        private static MLTAscendDbContext _db = new MLTAscendDbContext();
+
+        public dom.Prediction GetPredictionByTicker(string ticker)
+        {
+            return _db.Predictions.FirstOrDefault(m => m.Ticker == ticker);
```

Comment:
Maybe swap this to be "LastOrDefault" instead just because then we'd be grabbing the most recent prediction that was made for that ticker

Reply:
Fixed the datetime and getting the first one boss